.storybook/preview.js

```diff
@@ -2,13 +2,54 @@ import '@axa-ch/patterns-library-polyfill';
 import { addParameters } from '@storybook/html';
 import { withHTML } from '@whitespace/storybook-addon-html/html';
 
+const customViewports = {
+  xs: {
```

Comment:
Can we probably rename those or add a descriptor?

Looks a bit weird from a users perspective

![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/8248662/96458714-8c2e5700-1221-11eb-8e26-01b1a84c2a39.png)


Something like this would imho be better

Reply:
good idea

Reply:
But I will use extra small, small, medium, extra large and extra extra large due to our definition of breakpoint.
```$breakpoint-xs: 576px;
$breakpoint-sm: 768px;
$breakpoint-md: 992px;
$breakpoint-xl: 1200px;
$breakpoint-xxl: 1440px;```

Reply:
done

Reply:
i hate it but i will live with it, thx